### PR TITLE
Translation key dynamic cleaning

### PR DESCRIPTION
**Pull request by EstherDarkish**
Moves dynamically built translation keys away from the en.json and fr.json files and into new files in the dynamic directory.

### CODE REVIEW

File: src/translations/dynamic/errors-locale-en.js
Comment (Tristan-WorkGH) [inline src/translations/dynamic/errors-locale-en.js:1]: Why a JS with only static const and not a json like other translation files?

File: src/translations/dynamic/errors-locale-fr.js
Comment (Tristan-WorkGH) [inline src/translations/dynamic/errors-locale-fr.js:1]: Why a JS with only static const and not a json like other translation files?

File: src/translations/dynamic/menu-locale-en.js
Comment (Tristan-WorkGH) [inline src/translations/dynamic/menu-locale-en.js:1]: Why a JS with only static const and not a json like other translation files?

File: src/translations/dynamic/menu-locale-fr.js
Comment (Tristan-WorkGH) [inline src/translations/dynamic/menu-locale-fr.js:1]: Why a JS with only static const and not a json like other translation files?

**Review by Tristan-WorkGH — CHANGES_REQUESTED** [review]

File: src/translations/dynamic/errors-locale-en.js
Comment (EstherDarkish) [reply src/translations/dynamic/errors-locale-en.js:1]: Ok, renamed to .ts

File: src/translations/dynamic/errors-locale-fr.js
Comment (EstherDarkish) [reply src/translations/dynamic/errors-locale-fr.js:1]: Ok, renamed to .ts

File: src/translations/dynamic/menu-locale-en.js
Comment (EstherDarkish) [reply src/translations/dynamic/menu-locale-en.js:1]: Ok, renamed to .ts

File: src/translations/dynamic/menu-locale-fr.js
Comment (EstherDarkish) [reply src/translations/dynamic/menu-locale-fr.js:1]: Ok, renamed to .ts

**Review by Tristan-WorkGH — APPROVED** [review]

**Pull request merged**